cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
The `tests.py` file is empty. Consider adding test cases to verify the functionality of your `Movie` model and `MovieSerializer`. This will help maintain code quality and reliability.